workflows/mag.nf

```diff
@@ -19,8 +19,7 @@ if(hasExtension(params.input, "csv")){
                 if (row.size() == 5) {
                     if (row.long_reads) hybrid = true
                 } else {
-                    log.error "Input samplesheet contains row with ${row.size()} column(s). Expects 5."
-                    System.exit(1)
+                    Nextflow.error("Input samplesheet contains row with ${row.size()} column(s). Expects 5.")
```

Comment:
According to https://github.com/nf-core/sarek/pull/982#discussion_r1154287923 it should be sufficient to use
```suggestion
                    error("Input samplesheet contains row with ${row.size()} column(s). Expects 5.")
```

Reply:
@jfy133 It seems there was a misunderstanding, all was fine except this change here.
I copy the comment that I linked above, because it seems like you didnt have the time to check it out:

> The import and qualification with nextflow.error prefix are only necessary in Groovy classes pulled into the classpath via inclusion in the lib directory. Inside nextflow files such as main.nf, or workflows/sarek.nf, the error and exit methods are always in scope, so we can remove the import statement and change all occurrences of Nextflow.error to error.

Reply:
Yup, sorry, once I requested review I started multitasking :facepalm: . I should've replaced everything to just error again. I couldn't modify the groovy class because it's in teh template so got a linting fail...